shell/platform/tizen/tizen_vsync_waiter.cc

```diff
@@ -175,14 +162,14 @@ void TdmClient::VblankCallback(tdm_client_vblank* vblank,
                                unsigned int tv_sec,
                                unsigned int tv_usec,
                                void* user_data) {
-  TdmClient* client = reinterpret_cast<TdmClient*>(user_data);
-  FT_ASSERT(client != nullptr);
-  std::lock_guard<std::mutex> lock(client->engine_mutex_);
-  if (client->engine_) {
+  TdmClient* self = reinterpret_cast<TdmClient*>(user_data);
```

Comment:
I know that it can be replaced with auto in this cast situation.

Reply:
I changed but please note that it's not mandatory.